charts/rcon-web-admin/README.md

```diff
@@ -0,0 +1,61 @@
+# RCON Web Admin
+
+RCON Web Admin is a powerful web interface to control your RCON servers. All servers with RCON support should work.
+
+This application can install and run on a server, Raspberry Pi, or other devices which are online 24/7.
+
+Out of the box, RCON Web Admin can check users for high ping, VAC status, or filter the chat for you around the clock.
+
+## Introduction
+
+This chart creates a single RCON Web Admin Pod, plus Services for HTTP and WebSocket traffic.
+
+## Prerequisites
+
+- 512 MB of RAM
+- Kubernetes 1.4+ with Beta APIs enabled
```

Comment:
I only tested with v1.23.4+k3s1 and kind v1.23.1, so I have no clue which versions are supported :)

Reply:
Yeah 1.4 seems ancient. I don't mind leaving it or going with 1.23 since that's confirmed.